components/treeMap.vue

```diff
@@ -2,20 +2,30 @@
     <div>
         <svg
             id="treeMapViz"
-            :viewBox='[0, 0, 100, 100]'
-            class="viewBoxStyle" >
+            :viewBox='[0, 0, width, height]'
+            class="viewBoxStyle" 
+            >
             <g 
                 v-for="(leaf, index) in this.treeMapLeaves"
                     :key="index"
                     :transform="'translate(' + leaf.x0 + ',' + leaf.y0 +')'"
                     >
+                <text class="viewBoxStyle"> 
+                    {{ titleText(leaf) }}
+                </text>
                 <rect
                     :id="index+'leaf'"
-                    width="20%"
-                    height="20%"
-                    class="rectStyle"
-                >
+                    :width="leaf.x1 - leaf.x0"
+                    :height="leaf.y1 - leaf.y0"
+                    class="rectStyle">
                 </rect>
+                <circle
+                    :key="index"
+                    :cx="circleXPos(leaf)"
+                    :cy="circleYPos(leaf)"
+                    :r=10>
+                </circle>
```

Comment:
Circle for what?
Use `v-for` directive to render the users inside this leaf.
For instance: `v-for="user in leaf.data.user" ...`